### ISSUE TITLE
Notebook diff: fully unaccassible

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#105922]
https://github.com/microsoft/vscode/issues/105770

1. Open the my-work.github-issues file
2. Make some changes to it and open it in the SCM changes view
3. Try the `F7` or `shift + F7` experience to navigate through differences in an Accessible way (screen reader would read what is changed, you can try it with a regular editor). Does not work.

Due to this the Notebook diff is fully unaccassible. Adding important since this is a basic feature we have to support.

**Comment by isidorn** [comment]
I think you should just enable `F7` and `Shift + F7` commands to switch to plain json editor and to invoke the regular F7 experience across it.

**Comment by isidorn** [comment]
Alternative is that when the screen reader is detected we always open in it plain json view

**Comment by rebornix** [comment]
I pushed a change to always show text diff editor as suggested. The UI is not finalized yet so we will make it accessible once it's more polished.

**Comment by isidorn** [comment]
Sounds good, thanks for doing that!
